packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -146,6 +146,21 @@ class ScrollableBase extends Component {
 		 */
 		focusableScrollbar: PropTypes.bool,
 
+		/**
+		 * Specifies overscroll effects shows on which type of inputs.
+		 *
+		 * @type {String[]}
+		 * @default null
+		 * @private
+		 */
+		overscrollEffectOn: PropTypes.shape({
+			arrowKey: PropTypes.bool,
+			drag: PropTypes.bool,
+			pageKey: PropTypes.bool,
+			scrollbarButton: PropTypes.bool,
+			wheel: PropTypes.bool
```

Comment:
How about omitting `drag` and `wheel` here because those properties are not used in moonstone/Scrolllable? And we could merge `overscrollEffectOn` in moonstone into `overscrollEffectOn` default object in ui.

Reply:
As you pointed out, `drag` and `wheel` (in JS version) values are not used in `moonstone`. My intention is to expose all possible features explicitly. Also, if we define all values in `moonstone`, we don't need to merge `prop` objects and `ui/Scrollable` can simply read prop value without any modification or assigning to the local(this) variable. Could you share your opinion?

Reply:
I got your point.